code/modules/mob/living/organs.dm

```diff
@@ -0,0 +1,28 @@
+/mob/living
+	var/list/internal_organs = list()
+	var/list/organs = list()
+	var/list/organs_by_name = list() // map organ names to organs
+	var/list/internal_organs_by_name = list() // so internal organs have less ickiness too
```

Comment:
This will also give these vars to silicons

Reply:
Correct. They don't have any use on Silicons, but they are needed on simple_mobs, and it's the latest common branch.